src/metabase/public_settings.clj

```diff
@@ -683,3 +683,9 @@
   (deferred-tru "Prefix for upload table names")
   :visibility   :authenticated
   :type         :string)
+
+(defsetting bcc-enabled?
+  (deferred-tru "Whether or not bcc emails are enabled, default behavior is that it is")
+  :visibility :authenticated
+  :type       :boolean
+  :default    true)
```

Comment:
nit: For the sake of organization let's try to move away from using `metabase.public-settings` as a catch-all settings namespace. It was originally for _public_ settings specifically but it's become polluted with lots of different things.

Since settings are namespace-agnostic, they can go anywhere without changing behavior. This one might make the most sense in `metabase.email` near all the other email-related settings?